src/hooks/useSafeTokenAllocation.ts

```diff
@@ -126,30 +142,26 @@ const fetchTokenBalance = async (chainId: string, safeAddress: string): Promise<
  * The Safe token allocation is equal to the voting power.
  * It is computed by adding all vested tokens - claimed tokens + token balance
  */
-const useSafeTokenAllocation = (): [BigNumber | undefined, boolean] => {
+export const useSafeTokenBalance = (allocationData?: Vesting[]): AsyncResult<BigNumber> => {
```

Comment:
```suggestion
export const useSafeVotingPower = (allocationData?: Vesting[]): AsyncResult<BigNumber> => {
```

Should we rename it to `useSafeVotingPower`?
`useSafeTokenBalance` sounds to me as if it would just fetch the current balance in a hook.

Reply:
I've renamed it accordingly in ba34f38bd794ba7d4de458ff6aca8551b5d101d4.